site/docs/guides/basic-tutorial.md

```diff
@@ -635,8 +635,8 @@ The `useNode` hook accepts a collector function which can be used to retrieve st
 // components/user/Text.js
 export const Text = ({text, fontSize}) => {
   const { connectors: {connect, drag}, hasSelectedNode, hasDraggedNode, actions: {setProp} } = useNode((state) => ({
-    hasSelectedNode: state.events.selected.size > 0,
-    hasDraggedNode: state.events.dragged.size > 0
+    selected: state.events.selected,
```

Comment:
this is is not going to work unless you rename the `hasSelectedNode` on the line above